setup.py

```diff
@@ -52,7 +52,7 @@ def write_version_py(filename='VERSION'):
         # that you indicate whether you support Python 2, Python 3 or both.
         'Programming Language :: Python :: 2.7',
     ],
-    package_data={'pytripgui': ['res/*']},
+    package_data={'pytripgui': ['res/*', './pytripgui/view/*.ui']},
```

Comment:
Shouldn't it be:
```
package_data={'pytripgui': ['res/*', './view/*.ui']},
```

Read more here https://docs.python.org/3.7/distutils/setupscript.html#installing-package-data

Reply:
right